### ISSUE TITLE
Binary Sensors not updating motion

### ISSUE THREAD

**Issue by RichMawdsley1** [elad-bar/ha-blueiris#72]
Hi, setting up for the first time but the binary sensors are not updating from the mqtt status.

Confirmed the mqtt event is firing ok both from BI and manually, and can see the below coming in:

Topic:  BlueIris/Office/Status
Message:  { "type": "Connectivity", "trigger": "ON" }

(Where ‘Office’ is the Camera name)

However as stated the binary motion sensor for the camera in HA never updates.  How are the mqtt events being picked up by HA and turned into the binary? Am I supposed to have automations configured? Any assistance as to what is going wrong would be greatly appreciated.

**Comment by robotictacos** [comment]
I am also interested in this issue/solution.  I'm in an identical situation with my setup, and also don't see the motion sensor ever fire.  I also don't see any blueiris.advanced.yaml in my config folder when I select the option to generate the configuration files.  Additionally, I do not see any place that I could give this integration the MQTT login information so that it could even register/poll the topics.

**Comment by RichMawdsley1** [comment]
Yeah same.  I've tried all sorts of combinations now, MQTT is sending fine and being received, but whatever is supposed to be in the middle of the MQTT service and the Binary Sensors isn't working / does not exist.

**Comment by elad-bar** [comment]
can you please share a screenshot of the options configuration of the integration?

thanks

**Comment by RichMawdsley1** [comment]
Sure thing. Attached. 

![CE50BAA3-B1A2-43E4-BAD1-61BAA5A1F35F](https://user-images.githubusercontent.com/26411043/86486659-c13a9a80-bd53-11ea-8f0c-69d8bfe6d966.png)

**Comment by elad-bar** [comment]
Ok, more questions to understand the issue,
Did you set the topic name by yourself or used preservered variables of BI (as in the readme)?
Can you please change the log level of component to debug, restart HA and after few minutes (which the message is being sent through MQTT)?

Thanks

**Comment by RichMawdsley1** [comment]
In gathering the info.. you've led me to the problem!

The MQTT Message is where I've been going wrong.  I did not select in your Readme the 'Blue Iris Manual' when I set it up simply because I read it like that was how to setup BI itself and a BI manual nothing to do with this.  So I instead found the images of the config in the subdir on the wiki (I'm assuming @robotictacos did this too) and I had it setup like this:

https://github.com/elad-bar/ha-blueiris/blob/master/docs/images/bi-alerts-settings.png

Here, it's got 'Connectivity' in the title, and not &Type.  Changing this to &Type as per the screenshot the in 'Blue Iris Manual' it works straight away, and I can see it's passing through "MOTION_AB" as the type when it's triggered, and the binary sensors are updating.

Final note for anyone else who comes across this and perhaps yourself too @elad-bar , it does not work if you manually trigger the camera, a different &Type is passed and the binary sensor doesn't update.  Has to be a natural motion trigger.

Thanks for the help and great work on this as a whole @elad-bar .
